live-examples/js-examples/set-prototype-add.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var set1 = new Set();
+var object1 = new Object();
+
+set1.add(42);
+set1.add(object1).add('forty two');
+// chainable
+
+console.log(set1);
+// expected output: [object Set]
```

Comment:
I suppose (since we had space) it was to show that a `Set` was an `object`. But on reflection it does feel like it's not really required. I'll remove it.